### ISSUE TITLE
Drag and drop existing Apps over existing apps to make 'Services' of that App

### ISSUE THREAD

**Issue by tnaseem** [wavebox/waveboxapp#812]
* Wavebox Version: 4.2
* Operating System & Version: Windows 10

**Is your feature request related to a problem? Please describe.**
No

**Is your feature request related to an account type?**
No

**Describe the solution you'd like**
Great addition in the new version to allow the addition of other Services into an App. I'd like to suggest to allow the ability to move existing apps over too, either by dragging and dropping over the app, or dragging into the top toolbar.

Or, maybe by simply having an option for the app in its context menu to move it as a 'service' for another app?

**Comment by CheechGe** [comment]
Agree.

I hadn't sen your request and added the same feature request shortly after.  I've closed mine in deference to yours.

**Comment by Thomas101** [comment]
Yeah I agree this would be a good addition.

Each account down the side lives in its own sandbox though (for example you can have two Gmail accounts both entirely separate for each other). When moving a service from one account to another it would mean it would lose its session including things like logins.

Would this be a problem for what you're trying to do, or is there anything that you think we can do help communicate this before moving things around?

**Comment by CheechGe** [comment]
If you mean that in the process of doing the D&D from one account to another it dropped the connection I wouldn't have any problem with that.  I also would not have a problem signing on or re-entering my credentials as part of the process.  I didn't think it could be done in a totally non-disruptive way.

As far as a message is concerned, you could warn that the service would be broken and what would be necessary to complete the process.

**Comment by tnaseem** [comment]
Yeah. Same answer from me. It'd be a very minor 'annoyance' and better than having to go through every app, removing them and re-adding as 'Services'.

**Comment by mathieutu** [comment]
Should it be possible to leave them separated/sandboxed by default? There is already the option in settings for each service. So no need to loose session and reconnect in theses cases. (As, for me, the services will stay sandboxed, it's just about grouping services in the UI)

**Comment by Thomas101** [comment]
@mathieutu something along the lines of this - from when you add multiple of the same service to an existing account?
![screenshot 2018-12-10 at 15 01 15](https://user-images.githubusercontent.com/103586/49740606-81512580-fc8c-11e8-9ebd-055d7e687fd7.png)

**Comment by mathieutu** [comment]
Haven't seen this screen before, but it seems to be perfect for this usage, with a proper mention that "it will destroy your actual session" etc...
